<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdBlockAddSubscriptionDialog</class>
 <widget class="QDialog" name="AdBlockAddSubscriptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>145</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_cbUsePredefined">
       <property name="text">
        <string>Use predefined subscription</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_cmbPresets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_lblTitle">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="buddy">
      <cstring>m_txtTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_txtTitle">
     <property name="placeholderText">
      <string>Title of subscription</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_lblUrl">
     <property name="text">
      <string>URL</string>
     </property>
     <property name="buddy">
      <cstring>m_txtUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_txtUrl">
     <property name="placeholderText">
      <string>Absolute URL to online subscription file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_cbUsePredefined</tabstop>
  <tabstop>m_cmbPresets</tabstop>
  <tabstop>m_txtTitle</tabstop>
  <tabstop>m_txtUrl</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdBlockAddSubscriptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdBlockAddSubscriptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
